dev/ts/shared/main.ts

```diff
@@ -165,12 +165,9 @@ function collapseBpmnElement(bpmnElementId: string): void {
 }
 
 // callback function for opening | dropping the file to be loaded
-function readAndLoadFile(f: File): void {
-  const reader = new FileReader();
-  reader.addEventListener('load', () => {
-    loadBpmn(reader.result as string);
-  });
-  reader.readAsText(f);
+async function readAndLoadFile(file: File): Promise<void> {
```

Comment:
**thought:**  I don't understand why we need this change. I will check locally.
I thought we managed all "promises" related problems.

Reply:
Because the rule `unicorn/prefer-blob-reading-methods`, force use to replace `FileReader#readAsText(blob)` by `Blob#text()`; and `Blob#text()` returns a `Promise<string>`.